dxr/templates/file.html

```diff
@@ -12,19 +12,24 @@
   <link href="{{ url_for('.static', filename='css/tree-selector.css') }}" rel="stylesheet" type="text/css" media="screen" />
 {% endblock %}
 
-{% block content %}
-  <div class="breadcrumbs">
-    {%- for path, name in paths_and_names -%}
-      {%- if not loop.first or loop.last -%}
-        <span class="path-separator">/</span><a href="{{ www_root }}{{ path }}">{{ name }}</a>
-      {%- else -%}
-        <a href="{{ www_root }}{{ path }}">{{ name }}</a>
-      {%- endif -%}
-    {%- endfor -%}
-  </div>
-
-  {{ tree_menu(tree_tuples, tree) }}
+{% block nav_bar %}
```

Comment:
Makes sense. I'll do that. Except it will be needed in search.html (related to the other comment above). 

Also, I haven't been able to figure out how to get the error.html page to show up. I couldn't find where it gets referenced in the code, or where the template gets rendered.